view.js

```diff
@@ -175,10 +182,10 @@ const tree = (listItem) => {
 
 const showParent = (listItem) => {
   if (listItem.section.subpath === fileName) return true;
-
-  const hasText = /[a-zA-Z0-9]/g.test(
-    listItem.text.replace(/\[\[[^\]]+\]\]/g, "")
-  );
+  const cleanText = listItem.text
```

Comment:
Why did you add cleanText? Where is it used?

Reply:
I was using it for debugging `console.log({text: listItem.text, cleanText})`...
Easier to see what happens when messing around with regex replaces - it's only used in `const hasText` next statement.

Reply:
My apologies, I was looking at it on the phone and didn't read it quite well. this looks perfect!